js-envs-test-kit/src/main/scala/org/scalajs/jsenv/test/kit/TestKit.scala

```diff
@@ -9,22 +9,64 @@ import java.util.concurrent.Executors
 import org.scalajs.io.MemVirtualBinaryFile
 import org.scalajs.jsenv._
 
+/** TestKit is a utility class to simplify testing of [[JSEnv]]s
```

Comment:
Add a period at the end of the line so that Scaladoc knows where the short description ends.